activestorage/CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
+*   Allow variants to stay in the now supported `WebP` format during processing
+
+    Allow ActiveStorage Variants to stay in the now supported `WebP` format
+    during processing, instead of being converted to the fallback PNG format.
+    Done by adding `image/webp` to the list of known web image content types.
+
+    *Jeroen van Haperen*
+
```

Comment:
WebP variant support is new, so no need for a changelog entry.

Reply:
Should I add a commit to remove the full changelog addition? (you selected some lines from the changelog addition I did, so double checking)